SwanCustomEnvironments/swancustomenvironments/scripts/builders/accpy.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+# Activate Rust in order to recognize the uv command
+. "$HOME/.cargo/env"
```

Comment:
Why this? When is this file created?

Reply:
This file is generated automatically after installing UV and this line is appended to .bashrc. So everytime you open a terminal, you are able to use `uv` commands. As we just need to access `uv` for installing the requirements for accpy, we get this here